test/modules/aws-security-group-integration.ts

```diff
@@ -57,7 +57,7 @@ describe('Security Group Integration Testing', () => {
 
   it('installs the security group module', install(modules));
 
-  it(
+  /*it(
```

Comment:
These tests are going to be restored before merging, right?

Reply:
Sure ! I just commented to accelerate the testing, but i will readd . The way of mapping in the entity and the whole workflow makes sense to you ?